### PR TITLE
[docs] add MeshNetSimulator

### PR DESCRIPTION
**Pull request by mwarning**

### CODE REVIEW

**Comment by mwarning** [comment]
@nemesisdesign any improvements needed?

File: docs/source/implementations.rst [hunk 1]
```diff
@@ -116,6 +116,15 @@ netjsongraph.js
 library based on the popular d3.js visualization framework which can be used to
 visualize NetJSON ``NetworkGraph`` objects.
 
+MeshNetSimulator
+^^^^^^^^^^^^^^^
```
Comment (nemesisdesign) [inline docs/source/implementations.rst]: just add one more `^` under the `r` otherwise python-sphinx may not render it correctly, thanks

**Comment by nemesisdesign** [comment]
@mwarning sorry didn't notice the PR, thanks for pinging, left a small change request

**Comment by mwarning** [comment]
closed in favor of #64

**Pull request closed without merging**